test/install/test_no_ml.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) Fairlearn contributors.
+# Licensed under the MIT License.
+
+import pytest
+
+from fairlearn.adversarial import AdversarialFairness
+from fairlearn.adversarial._constants import _IMPORT_ERROR_MESSAGE
+
+
+@pytest.mark.parametrize("library", [None, 'torch', 'tensorflow'])
+def test_no_library(library):
+    expected_library = "torch or tensorflow" if library is None else library
```

Comment:
in a test, you can use `importorskip` from pytest.

Reply:
yeah no but this is to test the error message if none are installed. Quite silly though.